File: 02-Basic/JavaScript/03-primeNumber.js
function isPrimeNumber(number) {
    if (number <= 1) {
        return false; 
    }
    for (let i = 2; i <= number-1; i++) {
        if (number % i === 0) {
            return false; 
        }
        console.log(i)
    }

    return true;  
}

// Example usage
console.log(isPrimeNumber(41));  // Output: true
// console.log(isPrimeNumber(10)); // Output: false

File: 02-Basic/python/03-prime_number.py
def is_prime_number(number):
    if number <= 1:
        return False 
    
    for i in range(2, number - 1):
        if number % i == 0:
            return False  
    
    return True  


print(is_prime_number(5)) 
print(is_prime_number(79)) 
